backend/tests/controllers/auth/signup.test.js

```diff
@@ -0,0 +1,95 @@
+const request = require("supertest");
+
+const app = require("../../../app");
+
+describe("POST /signup", () => {
+  it("should be able to create an account", async () => {
+    const res = await request(app).post("/signup").send({
+      first_name: "Billy",
+      last_name: "Boy",
+      email: "bill123@gmail.com",
+      password: "pass",
+    });
+
+    expect(res.statusCode).toEqual(200);
+    expect(res.body.meta.code).toEqual(200);
+  });
```

Comment:
- keep in mind, we only not test the response code. We should test what the function really does. In this case we are checking _create account_ function, thus we should also test if there is data inserted after the api was executed. Add another `expect` for db checking.